en/core-libraries/caching.rst

```diff
@@ -131,6 +131,35 @@ refer to the class name using the following syntaxes:
     When using the FileEngine you might need to use the ``mask`` option to
     ensure cache files are made with the correct permissions.
 
+.. _cache-configuration-fallback:
+
+Configuring Cache Fallbacks
+---------------------------
+
+In the event that an engine is not available, such as the ``FileEngine`` trying
+to write to an unwritable folder or the ``RedisEngine`` failing to connect to
+Redis, the engine will fall back to the noop ``NullEngine`` and trigger a loggable
+error. This prevents the application from throwing an uncaught exception due to
+cache failure.
+
+You can configure Cache configurations to fall back to a specified config using
+the ``fallback`` configuration key::
+
+    Cache::config('redis', [
+        'className' => 'Redis',
+        'duration' => '+1 hours',
+        'prefix' => 'cake_redis_',
+        'host' => '127.0.0.1',
+        'port' => 6379,
+        'fallback' => 'default',
+    ]);
+
+If the Redis server unexpectedly failed, writing to the ``redis`` cache
+configuration would fall back to writing to the ``default`` cache configuration.
+If writing to the ``default`` cache configuration *also* failed in this scenario, the
+engine would fall back once again to the ``NullEngine`` and prevent the application
+from throwing an uncaught exception.
```

Comment:
Could you include a `.. versionadded:: 3.5.0` admonition here for folks using older versions.